components/partials/fee-discounts/overview.vue

```diff
@@ -219,6 +273,22 @@ export default Vue.extend({
       .finally(() => {
         this.status.setIdle()
       })
+  },
+
+  methods: {
+    handleScroll(e: Event) {
+      const target = e.target as HTMLElement
+      const offset = target.scrollLeft
+      const viewWidth = target.clientWidth
+      const total = target.scrollWidth - viewWidth
+      this.slideIndex = Math.round((offset / total) * 2)
+    }
   }
 })
 </script>
+
+<style lang="scss" scoped>
+.fee-discounts-statistic {
```

Comment:
Let's move this class to tailwind the `tailwind.config.js` file, for [example](https://v2.tailwindcss.com/docs/flex#flex-values).